### PR TITLE
Switch to Lightbend organisation and license

### PR DESCRIPTION
**Pull request by ennru**
## Purpose

Adapt to the home:
- update links in readme
- include Lightbend in the license
- change the organisation name to "com.lightbend.akka"

### CODE REVIEW

File: project/ProjectAutoPlugin.scala [hunk 1]
```diff
@@ -75,7 +79,13 @@ object ProjectAutoPlugin extends AutoPlugin {
     // show full stack traces and test case durations
     testOptions in Test += Tests.Argument("-oDF"),
 
-    headerLicense := Some(ALv2("2018", "Dennis Vriend")),
+    headerLicense := Some(
+      HeaderLicense.Custom(
+        """|Copyright (C) 2014 - 2019 Dennis Vriend <https://github.com/dnvriend>
+           |Copyright (C) 2019 - 2019 Lightbend Inc. <https://www.lightbend.com>
```
Comment (renatocaval) [inline project/ProjectAutoPlugin.scala]:
```suggestion
           |Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
```

I don't think we need a range 2019 - 2019. We can add the range 2019 - 2020 in a few days. ;-)

**Comment by ennru** [comment]
All builds fail. Do you see why that is? @renatocaval

**Comment by renatocaval** [comment]
Hmm, that's strange. It seems that none of the DBs where deployed But you can see [here](https://travis-ci.com/akka/akka-persistence-jdbc/jobs/263018698#L688-L691) that they did bootstrap.

**Comment by ennru** [comment]
The builds on `travis.com` fail, but the builds on `travis.org` succeed.
With https://github.com/akka/akka-persistence-jdbc/pull/256 I managed to swap that around...

**Comment by ennru** [comment]
Ok, for `travis.org` there are environment settings defined, they are not for `travis.com`.
image

**Pull request merged**